modules/role/manifests/st2express.pp

```diff
@@ -3,9 +3,17 @@
 
   include ::profile::infrastructure
   include ::profile::st2server
+  include ::profile::auth_backend_pam
   include ::profile::users
   include ::st2migrations
 
+  st2::helper::auth_manager {
```

Comment:
st2workroom_test is broken by this block.

See http://st2build002:8080/#/history/561f35d382fb9b66ce57550f

```
Error: This Block Expression has no effect. A value-producing expression without other effect may only be placed last in a block/sequence at /opt/puppet/environments/production/modules/role/manifests/st2express.pp:10:3
Error: This Name has no effect. A value-producing expression without other effect may only be placed last in a block/sequence at /opt/puppet/environments/production/modules/role/manifests/st2express.pp:10:3
```